.circleci/config.yml

```diff
@@ -30,7 +30,7 @@ jobs:
           command: |
             docker-compose up -d postgres
             sleep 3
-            docker-compose run compile tools/postgres/diff_schema.sh tools/postgres/schema.sql "conf/evolutions/*.sql"
+            docker-compose run compile tools/postgres/diff_schema.js tools/postgres/schema.sql "conf/evolutions/*.sql"
```

Comment:
This should simply run one step later, after `yarn install --frozen-lockfile`, so that all modules are installed.